### PR TITLE
Update documentation how to build and install fissile

### PR DESCRIPTION
**Pull request by zreigz**
Documentation describe how to build or install fissile

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,36 @@ Fissile converts existing BOSH dev releases into docker images.
...
 
+## Build and Install
+Building fissile needs Go 1.4 or higher. You can download it from [website](https://golang.org/doc/install)
+   
+### Build procedure
+Execute the following commands to compile fissile
+   
+```
+$ cd $GOPATH
+$ mkdir -p src
```
Comment (alexjh) [inline README.md]: `mkdir -p` creates parent dirs as needed, so only `mkdir -p src/github.com/hpcloud` is required.

File: README.md [hunk 2]
```diff
@@ -6,6 +6,36 @@ Fissile converts existing BOSH dev releases into docker images.
...
+$ mkdir -p src/github.com/hpcloud
+$ cd src/github.com/hpcloud
+$ git clone https://github.com/hpcloud/fissile.git
+$ cd fissile
+$ git submodule sync --recursive
+$ git submodule update --init  --recursive
+$ make all
+```
+
+Depends on your architecture you can use the fissile binary files from those directories:
```
Comment (andreas-kupries) [inline README.md]: `Depends` => `Depending`

Comment (zreigz) [inline README.md]: Done

File: README.md (hunk 1 above)
Comment (zreigz) [inline README.md]: Done

**Comment by zreigz** [comment]
PTAL

File: README.md [hunk 3]
```diff
@@ -6,6 +6,34 @@ Fissile converts existing BOSH dev releases into docker images.
 It does this using just the releases, without a BOSH deployment, CPIs, or a BOSH
 agent.
 
+## Build and Install
+Building fissile needs Go 1.4 or higher. You can download it from [website](https://golang.org/doc/install)
```
Comment (alexjh) [inline README.md]: Final nitpick from me, `website` is too generic. How about 'the Golang website'?

PS. Thanks for adding docs, we appreciate it!

Comment (zreigz) [inline README.md]: Done

**Pull request merged**